main.py

```diff
@@ -23,102 +27,38 @@
+parser.add_argument('--mailtm',action='store_true',default=MAILTM_DISABLED,required=False,help=MAILTM_DESCRIPTION,dest='mailtm')
+args = parser.parse_args()
 # END TEST
 
-def solveCaptcha(driver):
-    # Logic to click through the reCaptcha to the Audio Challenge, download the challenge mp3 file, run it through the audioToText function, and send answer
-    googleClass = driver.find_elements_by_class_name(CAPTCHA_BOX)[0]
-    time.sleep(2)
-    outeriframe = googleClass.find_element_by_tag_name('iframe')
-    time.sleep(1)
-    outeriframe.click()
-    time.sleep(2)
-    allIframesLen = driver.find_elements_by_tag_name('iframe')
-    time.sleep(1)
-    audioBtnFound = False
-    audioBtnIndex = -1
-    for index in range(len(allIframesLen)):
-        driver.switch_to.default_content()
-        iframe = driver.find_elements_by_tag_name('iframe')[index]
-        driver.switch_to.frame(iframe)
-        driver.implicitly_wait(2)
-        try:
-            audioBtn = driver.find_element_by_id(RECAPTCHA_AUDIO_BUTTON) or driver.find_element_by_id(RECAPTCHA_ANCHOR)
-            audioBtn.click()
-            audioBtnFound = True
-            audioBtnIndex = index
-            break
-        except Exception as e:
-            pass
-    if audioBtnFound:
-        try:
-            while True:
-                href = driver.find_element_by_id(AUDIO_SOURCE).get_attribute('src')
-                response = requests.get(href, stream=True)
-                saveFile(response, CAPTCHA_MP3_FILENAME)
-                response = audioToText(CAPTCHA_MP3_FILENAME)
-                print(response)
-                driver.switch_to.default_content()
-                iframe = driver.find_elements_by_tag_name('iframe')[audioBtnIndex]
-                driver.switch_to.frame(iframe)
-                inputbtn = driver.find_element_by_id(AUDIO_RESPONSE)
-                inputbtn.send_keys(response)
-                inputbtn.send_keys(Keys.ENTER)
-                time.sleep(2)
-                errorMsg = driver.find_elements_by_class_name(AUDIO_ERROR_MESSAGE)[0]
-                if errorMsg.text == "" or errorMsg.value_of_css_property('display') == 'none':
-                    print("reCaptcha defeated!")
-                    break
-        except Exception as e:
-            print(e)
-            print('Oops, something happened. Check above this message for errors or check the chrome window to see if captcha locked you out...')
-    else:
-        print('Button not found. This should not happen.')
-
-    time.sleep(2)
-    driver.switch_to.default_content()
-
 def start_driver(random_city):
-    driver = webdriver.Chrome(ChromeDriverManager().install())
+    options = Options()
+    if (args.debug == DEBUG_DISABLED):
+        options.add_argument(f"user-agent={USER_AGENT}")
+        options.add_argument('disable-blink-features=AutomationControlled')
+        options.headless = True
+        driver = webdriver.Chrome(options=options)
```

Comment:
Looks like ChromeDriverManager isn't being used anymore - any specific reason or was that added in before this fork was created?

Reply:
That was added before. This pull has been through a few changes, I'll revise to use the ChromeDriverManager as it's generally easier